components/rst-reader/src/main.rs

```diff
@@ -12,33 +12,36 @@ use log::error;
 use std::{path::PathBuf,
           process};
 
-pub mod error;
+mod error;
+
+#[derive(Debug, Clone, Parser)]
+#[command(name = "Habitat Rst Reader",
+          about = "Introspection for the butterfly RST file.",
+          arg_required_else_help = true,
+          help_template = "{name}\n{about-section}\n{usage-heading} {usage}\n\n{all-args}")]
+struct RstReader {
+    #[arg(name = "FILE")]
+    file: String,
 
-fn main() {
+    #[arg(name = "STATS", short = 's', long = "stats")]
+    stats: bool,
+}
+
+fn main() -> error::Result<()> {
     env_logger::init();
-    let matches =
-        App::new("Habitat RST Reader").about("Introspection for the butterfly RST file")
-                                      .arg(Arg::with_name("FILE").required(true)
-                                                                 .index(1)
-                                                                 .help("Path to the RST file"))
```

Comment:
looks like we lose this arg specific help and for STATS. Is there a way to retain that?

Reply:
Oh!! Sorry had missed that! Fixed now.